### DISCUSSION TITLE
When a drug has multiple metabolites via an enzyme, how should it be set up in PKsim?

### DISCUSSION THREAD

**Discussion by noxavier** [Open-Systems-Pharmacology/Forum#1484] [Questions & Problems]
Suppose CYP3A4 metabolizes a drug to produce three metabolites M1, M2, and M3. I want to simulate the PK of M1-3 and have obtained the Km and Vmax of M1-3, but I have not done this before in PK-sim. Does anyone have a similar experience that can guide me?

Thanks!

**Comment by prvmalik** [comment]


**Reply by noxavier** [reply]
My understanding is that CYP3A4-1 and CYP3A4-2 are set in INDIVIDUAL, and the corresponding parameters are selected in the simulation. Can you see if I am setting this correctly? Also, I have a question, when I set two CYP3A4, should I halve the concentration of each one?
![individual](https://user-images.githubusercontent.com/109507330/227407399-9f8dbd56-e59e-4298-91d8-e16c0fb7c5c2.png)

![a27461c8c3091cd9aa92380157efa1c](https://user-images.githubusercontent.com/109507330/227407408-8bf9f021-952c-4595-aeeb-ca93c56db97f.png)

**Reply by noxavier** [reply]
Oh, sorry, I made a mistake in the screenshot, in the second picture, I meant to set CYP3A4-1 and 3A4-2 instead of 1A2 and 3A4. 
And if you suggest adjusting the Vmax, do you suggest setting the Km for the corresponding metabolites individually, or do you suggest setting the sum of the metabolites? 
Thank you for your help and advice. This project has not started yet, if you agree, I will summarise the problems I encountered and ask you for advice.

**Comment by Yuri05** [comment]
>A single enzyme can only create a single metabolite in PK-Sim so you will need two enzymes for two metabolites, and so on. If CYP3A4 happens to create two different metabolites, you will have to insert it twice, for instance. 

This is not quite true: **you can produce several metabolites for the same enzyme in PK-Sim**.

* Add enzyme in the individual (only once!)
![grafik](https://user-images.githubusercontent.com/25061876/227493936-d92b0e34-1ec0-4eb1-99e0-38dc7a698d6f.png)
* Add new metabolization process for this enzyme and **for every metabolite** in compound (you will need to enter your Km/Vmax for each process)
![grafik](https://user-images.githubusercontent.com/25061876/227495751-1e5aca77-7a0a-4983-abf0-513654e08daa.png)
* When creating a simulation: select the compounds and all metabolites which time courses should be plotted
![grafik](https://user-images.githubusercontent.com/25061876/227496153-31592ffb-d284-4934-908e-aae6d4596bfd.png)
* When it comes to the process mapping: PK-Sim suggests only 1 of created metabolization processes **as per default**. 
  ## Use the plus button to add a process for the same enzyme and other metabolites!
![grafik](https://user-images.githubusercontent.com/25061876/227496568-6ecb0445-b205-405a-aa59-97a4e60f287b.png)
* Will look like this at the end:
![grafik](https://user-images.githubusercontent.com/25061876/227497266-ff905a28-b60e-4b25-9431-56a65d2651e4.png)
* What you get in the simulation is then:
![grafik](https://user-images.githubusercontent.com/25061876/227497874-bf5076bc-0375-4605-9759-faeee42bb676.png)

s. the example project: 
[M1M2M3.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/11061073/M1M2M3.zip)
